cvat/apps/engine/ddln/inventory_client.py

```diff
@@ -31,21 +56,62 @@ def __init__(self, spreadsheet_id, credentials_file, scopes=None):
         self.spreadsheet_id = spreadsheet_id
 
     def record_sequence_completion(self, sequence_name: str, task_name: str, annotator: str, completion_date: dt.date):
+        row_index = self._get_row_index(sequence_name, task_name)
+        if row_index == -1:
+            raise ValueError("sequence {!r} for task {!r} is not found.".format(sequence_name, task_name))
         completion_date = "{:%d.%m.%Y}".format(completion_date)
-        row = [sequence_name, 'image', 'CVAT', task_name, completion_date, annotator]
+        row = [annotator, completion_date]
+        request = self._service.spreadsheets().values().update(
+            spreadsheetId=self.spreadsheet_id,
+            range='E{}:F{}'.format(row_index, row_index),
+            valueInputOption='USER_ENTERED',
+            body={
+                'values': [row]
+            }
+        )
+        response_data = request.execute()
+        return response_data['updatedRange']
+
+    def record_task_creation(self, task_name: str, sequence_annotator_pairs: List[Tuple[str, str]]):
```

Comment:
nit: a bit inconsistent that types are declared for some functions in this file, but not for others
also, a type like `List[Tuple[str, str]]` is not helpful for readability
better add a comment explaining the arguments